File: src/test/java/no/nav/syfo/testhelper/NarmesteLederGenerator.java
package no.nav.syfo.testhelper;

import no.nav.syfo.model.Naermesteleder;

import static no.nav.syfo.testhelper.UserConstants.LEDER_AKTORID;

public class NarmesteLederGenerator {

    private final Naermesteleder naermesteleder = new Naermesteleder()
            .naermesteLederAktoerId(LEDER_AKTORID);

    public Naermesteleder generateNarmesteLeder() {
        return naermesteleder;
    }
}

File: src/test/kotlin/no/nav/syfo/testhelper/NarmesteLederGenerator.kt
package no.nav.syfo.testhelper

import no.nav.syfo.model.Naermesteleder
import no.nav.syfo.testhelper.UserConstants.LEDER_AKTORID

class NarmesteLederGenerator {
    private val naermesteleder = Naermesteleder()
        .naermesteLederAktoerId(LEDER_AKTORID)

    fun generateNarmesteLeder(): Naermesteleder {
        return naermesteleder
    }
}
